src/main/java/ui/listpanel/ListPanel.java

```diff
@@ -35,19 +39,29 @@
 import backend.resource.TurboIssue;
 import filter.expression.Qualifier;
 
+
 public class ListPanel extends FilterPanel {
 
     private static final Logger logger = HTLog.get(ListPanel.class);
 
     private final UI ui;
     private final GUIController guiController;
-    private int issueCount;
+    private int issueCount = 0;
+    private int closedIssueCount = 0;
+    private int openIssueCount = 0;
 
     private final IssueListView listView;
     private final HashMap<Integer, Integer> issueCommentCounts = new HashMap<>();
     private final HashMap<Integer, Integer> issueNonSelfCommentCounts = new HashMap<>();
 
-    private Label totalLabel;
+
+
+    Text openIssueText;
```

Comment:
Same thing there. I think 1 blank line should be readable enough to separate groups of variables.